PlanningPoker.App/Models/Settings.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+
+namespace PlanningPoker.App.Models
+{
+    public class Settings : ISettings
+    {
+        //TODO: Replace with *.azurewebsites.net url after deploying backend to Azure
```

Comment:
AFAIK kommer vi ikke til at deploye til Azure. Ikke hvis vi kan slippe for det. Time will tell.